src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  width: $card-width;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 32px 16px 16px;
+  font-family: $font;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: all 0.3s;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin: 0;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $card-color;
+
+  }
+
+  &__code {
+    margin-top: 4px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &__raiting {
+    margin-top: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-comments {
```

Comment:
&-comments is not an element of the card__raiting. Move it outside the rating, please